### DISCUSSION TITLE
Dialog Trigger inside NavigationMenu

### DISCUSSION THREAD

**Discussion by sushantgwr87** [radix-ui/themes#294] [Q&A]
Hey, so I am using Navigation menu to create navbar and Dialog to show Modal.

Problem is coming I have a ModalTrigger button inside navbar, On Navbar open if someone clicks on modalTrigger button, navbar closes then modal opens and closes as well with the navbar close.

**Comment by triptu** [comment]
Radix Themes doesn't have Navigation Menu, so I assume your question is about Radix primitives? In which case, a [custom portal](https://www.radix-ui.com/primitives/docs/components/dialog#custom-portal-container) for the dialog should likely solve it.
